Source/Script/PowerShell/Tfs/Archive.ps1

```diff
@@ -0,0 +1,16 @@
+﻿Param
+(
+    $FtpUser,
+    [SecureString]
+    $FtpPassword,
+    $FtpHost
+)
+
+mkdir $(Agent.BuildDirectory)\zip;
+
+Add-Type -Assembly 'System.IO.Compression.FileSystem';
+
+[System.IO.Compression.ZipFile]::CreateFromDirectory("$(Agent.BuildDirectory)\artifacts\drop\", "$(Agent.BuildDirectory)\zip\$(Build.BuildNumber).zip\");
```

Comment:
This build number is orthogonal to the emulator version.
The Pj64 binaries still keep their metadata and manifest info.

Again, it's for ClickOnce packaging, and doesn't I terrero with the traditional installer or zip.